frontend/src/views/components/LayersList/Basemaps/constants.ts

```diff
@@ -0,0 +1,18 @@
+export const MAP_LABELS = ['dark', 'light', 'none'] as const;
+
+export const MAP_LABELS_OPTIONS = [
+  {
+    label: 'Dark labels',
```

Comment:
We need to translate the labels. Check the 'Strings inside constants' part of the translation on the Readme. We should turn MAP_LABELS_OPTIONS into a function that will regenerate whenever we change locales and add a t('') component from react/native for each label.

So this will be:

export const getMapLabelsOptions =  () => [
{
  label: t('Dark labels'),
  value: 'dark',
}, ...
];

And in the component:

const translation = useMemo(() => getMapLabelsOptions(), [locale]);